content/library/quick-references/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Quick reference
+slug: /develop/quick-reference
+---
+
+# Quick reference
+
+<TileContainer layout="list">
+
+<RefCard href="/develop/quick-reference/cheat-sheet">
+
+<h5>Cheatsheet</h5>
+
+A dense list of Streamlit commands with example syntax.
+
+</RefCard>
+
+<RefCard href="/develop/quick-reference/changelog">
+
+<h5>Changelog</h5>
+
+See how Streamlit has changed with each new version.
+
+</RefCard>
+
+<RefCard href="/develop/quick-reference/prerelease">
+
+<h5>Pre-release features</h5>
```

Comment:
Not a blocker for this PR, but this seems like a strange place to talk about how we handle pre-release features.

Reply:
Happy to think about a better location in follow ups. I just kept those top-level references together for now. I did also think about reworking and restructuring when and how we introduce nightly and experimental features, but that might get into some rewriting...